pkg/messenger/messager.go

```diff
@@ -73,31 +75,28 @@ func (m *Messenger) Start(ctx context.Context) error {
 
 		log.Println("Received message:", msg.LoggableID)
 
-		m.handleRequest(context.Background(), msg)
-
-		// Slow down a bit to avoid churning through messages and running
```

Comment:
I don't think we should remove the error logic in this PR. I think we really don't want to be churning through 100 million requests, churning out 100 million error responses, leaving the user with a hefty bill. Too many things can go wrong and lead to that situation IMO.

Reply:
fixed, brought it back